Quickref.md

```diff
@@ -48,17 +48,22 @@ Since MonadHold depends on MonadSample, any [S] function also runs in [H] contex
 [ ]   attachPromptlyDynWithMaybe :: (a -> b -> Maybe c) ->  Dynamic a -> Event b -> Event c
 
 -- Combine multiple Events
-[ ]   <>        :: Monoid a => Event a -> Event a -> Event a
+[ ]   <>        ::         Monoid a => Event a -> Event a -> Event a
+[ ]   unlessE   ::                     Event a -> Event b -> Event a
```

Comment:
Going back to our discussion before, perhaps we should call this `difference` or something in that vein, analogously to [Data.Map.difference](https://hackage.haskell.org/package/containers-0.5.8.1/docs/Data-Map-Strict.html#v:difference)